### PR TITLE
[OBSDEF-10894] add Alert for CRR - REPLICATION_EVENT_RETRY_LIMIT_EXCEEDED

### PR DESCRIPTION
**Pull request by chengmx1025**
## Purpose
[OBSDEF-10894](https://jira.cec.lab.emc.com/browse/OBSDEF-10894)
_List the changes for this PR_
  add new AlertType REPLICATION_EVENT_RETRY_LIMIT_EXCEEDED

## PR checklist
- [x] make test passed
- [x] make build passed
- [x] helm install <chart> passed
- [ ] vSphere7 make package deployments passed
- [ ] helm upgrade <chart> passed
- [ ] helm test <release_name> passed
- [ ] Passed - https://asd-ecs-jenkins.isus.emc.com/jenkins/view/ECS-Flex/job/charts-custom-ci/

## Testing
_Paste URL of charts-custom-ci job here_

_Paste the output of your helm install/vSphere7 deployment testing here_

### CODE REVIEW

**Review by eddiepavkovic — CHANGES_REQUESTED** [review]
The name has to change this is shown to customers.  We can't have the world bug

File: ecs-cluster/alerts/issues/12007_REPLICATION_BUG_DETECTED.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+- description: "Detected bugs during replication procedure, get more details from Bug-Stream."
+  name: "REPLICATION_BUG_DETECTED"
```
Comment (vgupta-mickey) [inline ecs-cluster/alerts/issues/12007_REPLICATION_BUG_DETECTED.yaml:2]: It should be problem specific

**Review by vgupta-mickey — CHANGES_REQUESTED** [review]

File: ecs-cluster/alerts/issues/12007_REPLICATION_BUG_DETECTED.yaml (hunk 1 above)
Comment (chengmx1025) [reply ecs-cluster/alerts/issues/12007_REPLICATION_BUG_DETECTED.yaml:2]: could I change the error code name to "UNKNOWN_REPLICATION_ERROR_DETECTED" ?

File: ecs-cluster/alerts/issues/12007_UNKNOWN_REPLICATION_ERROR_DETECTED.yaml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 - description: "Detected an unknown error during replication procedure"
-  name: "UNKNOWN_REPLICATION_ERROR"
+  name: "UNKNOWN_REPLICATION_ERROR_DETECTED"
```
Comment (eddiepavkovic) [inline ecs-cluster/alerts/issues/12007_UNKNOWN_REPLICATION_ERROR_DETECTED.yaml:2]: What does this mean?   How will customers know what remedies 
To perform if they hit this problem.  

We shouldn't be vague like this.  It's just going to have customers 
Frustrated and call dell.

**Review by eddiepavkovic — CHANGES_REQUESTED** [review]

File: ecs-cluster/alerts/issues/12007_UNKNOWN_REPLICATION_ERROR_DETECTED.yaml (hunk 2 above)
Comment (dingr5) [reply ecs-cluster/alerts/issues/12007_UNKNOWN_REPLICATION_ERROR_DETECTED.yaml:2]: I suppose we could modify the name but I think this Alert is send to Dell only channel.

**Review by dingr5 — APPROVED** [review]

**Comment by chengmx1025** [comment]
Hi @eddiepavkovic @vgupta-mickey,
I've renamed the Alert code to REPLICATION_EVENT_RETRY_LIMIT_EXCEEDED.
And also, this alert will be only visible to EMC OE team.

**Comment by eddiepavkovic** [comment]
Why was my review request removed.

**Comment by dingr5** [comment]
> Why was my review request removed.

Hi @eddiepavkovic , sorry for the delayed reply. We have already reply addressed the changed based on the comment and re-request for 6 days. So we thought there should be no more comments and we should be able to continue. Will double check with you next time.

**Pull request merged**